### ISSUE TITLE
A11y issues in Crane

### ISSUE THREAD

**Issue by dnfield** [material-components/material-components-flutter-gallery#135]
## Acceptance Criteria
* Images/buttons should be labeled.
* Lists should be a11y traversible (Currently, some items are not reachable by a11y navigation).
* Tabs probably should not have nested a11y buttons that do the same thing as selecting the tab.

/cc @goderbauer @vick08 @darrenaustin @willlarche
